data/json/items/comestibles/seed.json

```diff
@@ -10,9 +10,24 @@
         "price": 120,
         "charges": 4,
         "stack_size": 100,
-        "material": "fruit",
+        "material": "veggy",
```

Comment:
If `seed` can be either `fruit` or `veggy` it shouldn't be specified here at all